tests/test_workspace.py

```diff
@@ -746,3 +746,14 @@ class FooWorkspace(pyhf.Workspace):
 
     combined = FooWorkspace.combine(ws, new_ws)
     assert isinstance(combined, FooWorkspace)
+
+
+def test_sorted(workspace_factory):
+    ws = workspace_factory()
+    # force the first sample in each channel to be last
+    for channel in ws['channels']:
+        channel['samples'][0]['name'] = 'zzzzlast'
+
+    new_ws = pyhf.Workspace.sorted(ws)
+    for channel in ws['channels']:
```

Comment:
```suggestion
    for channel in new_ws['channels']:
```

@kratsg This should be `new_ws` yes? If so, then fixing this also get `pyflakes` happy